wordle-web/src/components/__tests__/HelloWorld.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from 'vitest'
+
+import { mount } from '@vue/test-utils'
+import HelloWorld from '../HelloWorld.vue'
+
+describe('HelloWorld', () => {
+  it('renders properly', () => {
+    const wrapper = mount(HelloWorld, { props: { msg: 'Hello Vitest' } })
+    expect(wrapper.text()).toContain('Hello Vitest')
+  })
+})
```

Comment:
Nice work on writing this test! I was trying to do the same thing in my project but i wasnt able to figure it out. Great job!

Reply:
Thanks!